libs/blockchain-watcher/feature-app/src/lib/util/IsEthAddress.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  registerDecorator,
+  ValidationOptions,
+  ValidationArguments,
+} from 'class-validator'
+import { stringHelper } from '../helper/stringHelper'
+
+export function IsEthAddress(validationOptions?: ValidationOptions) {
+  // turn off @typescript-eslint/ban-types because of object: Object
```

Comment:
I see the problem now. I think it's better to disable locally than turn off `@typescript-eslint/ban-types`, since it's really helpful in other situations.

Also, instead of `object: Object`, can you do `Record<string, unknown>`?

Reply:
I think disabling it locally would be better, but I'm not sure about the object type. ESLint suggest using `object: Record<string, unknown>`, but `class-validator` use `object: Object` in their document.

Reply:
I see, I think in this case `Record<string, unknown>` is acceptable, it's more accurate than `object: Object` and if there's no type error it should work fine.

Reply:
Just found that `Record<string, unknown>` cause type error in class decorator. Rollback to `object: Object`.

Reply:
Ah, now I understand. I see you already updated to disable locally, thanks!